src/commands/repeat.rs

```diff
@@ -1,35 +1,36 @@
 use serenity::{
-    client::Context,
-    framework::standard::{macros::command, CommandResult},
-    model::channel::Message,
+    client::Context, model::interactions::application_command::ApplicationCommandInteraction,
+    prelude::SerenityError,
 };
 use songbird::tracks::{LoopState, TrackHandle};
 
-use crate::{strings::NO_VOICE_CONNECTION, utils::send_simple_message};
+use crate::{strings::NO_VOICE_CONNECTION, utils::create_response};
 
-#[command("loop")]
-async fn repeat(ctx: &Context, msg: &Message) -> CommandResult {
-    let guild_id = msg.guild(&ctx.cache).await.unwrap().id;
+pub async fn repeat(
+    ctx: &Context,
+    interaction: &mut ApplicationCommandInteraction,
+) -> Result<(), SerenityError> {
+    let guild_id = interaction.guild_id.unwrap();
     let manager = songbird::get(ctx).await.unwrap();
 
     let call = match manager.get(guild_id) {
         Some(call) => call,
-        None => return send_simple_message(&ctx.http, msg, NO_VOICE_CONNECTION).await,
+        None => return create_response(&ctx.http, interaction, NO_VOICE_CONNECTION).await,
     };
 
     let handler = call.lock().await;
     let track = handler.queue().current().unwrap();
 
-    let was_looping = track.get_info().await?.loops == LoopState::Infinite;
+    let was_looping = track.get_info().await.unwrap().loops == LoopState::Infinite;
```

Comment:
Why removal of ?

Reply:
Same reason as above, can't convert the error to a SerenityError.